### ISSUE TITLE
Missing step in instructions?

### ISSUE THREAD

**Issue by leanderlindahl** [roots/bedrock#225]
Hello, thanks for sharing this project. 

I'm trying to install it "by the book":

I cloned the Bedrock repo to my local environment and set up my virtual host to point to the folder.

I ran composer install.

I  copied the .env.example to .env and provided the DB, ENV, HOME and SITEURL vars, but omitted the salts. 

When I try load the site at "my.site.dev" the url switches to "my.site.dev/wp/wp-admin/install.php" which looks good, but then I get connection timeout or "not available". I can however load load the readme.html so the setup seems to be working in itself.

So then I decided to try add the salts with wp dotenv generate salts (after installing the https://github.com/aaemnnosttv/wp-cli-dotenv-command) 

But I get 

Error: The site you have requested is not installed.
Run `wp core install`.

Any ideas where I went wrong?

**Comment by leanderlindahl** [comment]
Ok, so reading in #184 I discover salts are not needed on local install...
I still find the instructions slightly confusing, sorry if I'm being thick, but I'm unclear about what the sequence is
1. Clone bedrock to your machine
2. Execute composer install
3. Copy .env.example to .env and add your case specific settings (db name, dblogin etc)
4. Add a theme to themes folder
5. Configure your local machine (hosts, vhosts)
6. Log in at wp/wp-admin

What I don't understand is how/when the variables in .env will be transformed into a wp-config file. There is no command executed after the .env is created and given the db settings

**Comment by QWp6t** [comment]
They're not transformed into `wp-config.php`. They're parsed by DotEnv and then retrieved as environment variables in `application.php`.

https://github.com/roots/bedrock/blob/master/config/application.php

The Bedrock issue tracker is for ongoing Bedrock development, not for support. Please use https://discourse.roots.io/ for support requests.

**Comment by leanderlindahl** [comment]
QWp6t, thanks for your answer and I can see your point about this not being the right forum for support, but I still feel the instructions seem to be missing some step or steps, which is a legitimate issue for the project.

**Comment by leanderlindahl** [comment]
Perhaps the requirements need to be clarified if it is not the installation instructions? I have PHP 5.5 (5.5.29) and Composer installed and followed the installation instructions.

**Comment by swalkinshaw** [comment]
@leanderlindahl documentation can always be improved, but in this I don't see anything you've missed.

If you can get to `my.site.dev/wp/wp-admin/install.php` but then the install doesn't work due to a bad DB connection then something must be wrong with your DB/the credentials you used.

**Comment by leanderlindahl** [comment]
@swalkinshaw Thanks for taking the time to reply. I really like the entire Roots project and "suite" and it's a great resource and opportunity to pick up good coding and workflow practices. I use it whenever there's an opportunity.

Now to the actual issue: I know the DB credentials are valid. I run 30-40 local installs with the same credentials. And I know the virtual host i correctly configured because I can load loc.mysite.dev/readme.html... I even started to think perhaps the .env file doesn't have the right permissions and can't be read and put the entire project (including .env) to chmod 777, just to make sure.

I guess this is something we would have discovered right away if we were in the same room but obviously it can't be conveyed or identified in this format. I believe you when if you say it is working both in theory and in practice. But it doesn't for me. (And I've hade the entire stack with Trellis installed and working several times, so I'm not completely lost with this stuff).

My guess is that there is one piece of the puzzle somehow missing and that's why I suggested  that there might be some "implicit" knowledge that could be missing in the requirements or instructions. 

Following QWp6t suggestion I have continued the discussion here https://discourse.roots.io/t/missing-step-installing-bedrock/5542
